smart-hadoop-support/smart-hadoop-client-cdh-2.6/src/main/java/org/smartdata/hadoop/filesystem/SmartFileSystem.java

```diff
@@ -179,6 +187,16 @@ public FileStatus getFileStatus(Path f) throws IOException {
             oldStatus.getOwner(), oldStatus.getGroup(),
             oldStatus.isSymlink() ? oldStatus.getSymlink() : null, oldStatus.getPath());
       }
+    } else {
```

Comment:
What if oldStatus=null? This else code block will throw NPE when oldStatus=null.